index.html

```diff
@@ -419,5 +427,13 @@ <h2 class="form-header">Contact me</h2>
         </div>
       </section>
     </div>
+    <div class="overlay-menu hidden">
+      
+      <a href="#portfolio" style="display: block; color: white;">Portfolio</a>
+            <a href="#about" style="display: block; color: white;">About</a>
+            <a href="#contact" style="display: block; color: white;">Contact</a>
+    </div>
+
+    <script src="./main.js"></script>
```

Comment:
- Kindly make sure to match the Figma design as much as possible, which includes:
   - [ ] Using the proper font size indicated in the original design.
   - [ ] Implementing a "close button" on the top right corner of the mobile menu which functions to close the menu as per the design.
   - [ ] Editing the spacing between and around the mobile menu links to match the design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/231202649-7a2562c2-4df0-4559-b56f-339e53a2f0ac.png) | ![image](https://user-images.githubusercontent.com/62290545/231203073-12b10ec0-3193-4d2f-b45b-a9474e06b75f.png) |

Reply:
Fixed The Issue.